aca_preview/preview.py

```diff
@@ -648,21 +650,31 @@ def check_acq_p2(self):
         elif P2 < P2_lim + 1:
             self.add_message('warning', f'P2: {P2:.2f} less than {P2_lim + 1} for {obs_type}')
 
-    def check_bright_guide_for_ers(self, n_bright_req=3, bright_lim=9.0):
-        """Check for at least 3 guide stars brighter than 9th mag for ERs.
+    def check_guide_fraction(self):
+        """Check for sufficient guide star fractional count
 
         """
-        n_bright = np.count_nonzero(self.guides['mag'] < bright_lim)
-        if self.is_ER and n_bright < n_bright_req:
+        if self.is_ER and self.guide_count < 3.0:
+            self.add_message(
+                'critical',
+                f'ER guide count: only {self.guide_count} bright stars for mag')
+        if self.is_OR and self.guide_count < 4.0:
             self.add_message(
-                'critical', f'ER bright stars: only {n_bright} stars brighter than {bright_lim}')
+                'critical',
+                f'OR guide count: {self.guide_count} < 4.0')
 
-    def check_enough_guide_for_ers(self, n_required=8):
-        """Warn on ERs with fewer than n_required (8) guide stars.
+    def check_enough_guide_for_ers(self):
+        """Warn on ERs with fewer than required or desired number of guide stars.
 
         """
-        if self.is_ER and len(self.guides) < n_required:
-            self.add_message('critical', f'ER guide stars: only {len(self.guides)} stars')
+        if self.is_ER:
+            for n, category in ((6, 'critical'),
```

Comment:
Actually what probably makes sense here and is consistent with ORs is to just have one critical warning for `self.guide_count < 6.0` (where this is the version called with `count_9th=False`).  Then just cut this whole check.

The only question is precedent.  This standard may be higher than what we've been planning, not sure.

Reply:
OK.  That would simplify.